src/utils/render.ts

```diff
@@ -0,0 +1,21 @@
+import { color, Color } from 'd3-color';
```

Comment:
Maybe this file could be called utils/color? It doesn't do any rendering

Reply:
Done